presto-native-tests/src/test/java/com.facebook.presto.nativetests/TestOrderByQueries.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.nativetests;
+
+import com.facebook.presto.nativeworker.NativeQueryRunnerUtils;
+import com.facebook.presto.nativeworker.PrestoNativeQueryRunnerUtils;
+import com.facebook.presto.testing.QueryRunner;
+import com.facebook.presto.tests.AbstractTestOrderByQueries;
+import org.testng.annotations.Test;
+
+public class TestOrderByQueries
+        extends AbstractTestOrderByQueries
+{
+    private static final String storageFormat = "PARQUET";
+
+    @Override
+    protected QueryRunner createQueryRunner() throws Exception
```

Comment:
Do we plan to overload the expected query runner with the JavaQueryRunner as well ?

Reply:
No we do not plan to overload the expected query runner with javaQueryRunner because it does not account for the datatypes of the computed result when comparing it with the expected values. This limitation is [noted here](https://ibm.box.com/s/2ph2ttr3a2zqs97uu8ik6ie9xshww8sj).